### ISSUE TITLE
Country code AR does not exist in CountryIncomeThreshold for financial aid id 14815

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#4045]
I looked back through out correspondence with DEDP, and it looks like Argentina ("AR") has been missing since the beginning. 

https://sentry.io/mit-office-of-digital-learning/micromasters/issues/591089721/

```
Country code AR does not exist in CountryIncomeThreshold for financial aid id 14815
```

**Comment by pdpinch** [comment]
From DEDP: 

> Since Argentina is an upper-middle income country it looks like the threshold should be 50,000.

**Comment by pdpinch** [comment]
I had thought that this was just a matter of adding the data to the fixture at https://github.com/mitodl/micromasters/blob/master/financialaid/fixtures/country_income_threshold_data.json#L62 but it looks like this isn't the same as the final data in the database?

**Comment by amir-qayyum-khan** [comment]
@pdpinch migration loading thresholds from the file you suggested https://github.com/mitodl/micromasters/blob/master/financialaid/migrations/0016_load_country_income_thresholds.py#L18.

I dont know why data set is change on https://micromasters-rc.odl.mit.edu/admin/financialaid/countryincomethreshold/

May be we changed it manually? or there is separate country_income_threshold_data.json for production?

cc @mitodl/devops

**Comment by amir-qayyum-khan** [comment]
yes, there is a command to `sync_currency_exchange_rates`. but i do not update `CountryIncomeThreshold` table. It changes CurrencyExchangeRate
